README.md

```diff
@@ -19,7 +19,14 @@ services:
 
     volumes:
       - .:/app
-      - ~/.ssh:/home/application/.ssh
+      - ~/.ssh/config:/home/application/.ssh/config:ro
```

Comment:
i don't like having mapped that many volumes into my container and i don't think thats wise performance-wise

Reply:
Removed